### ISSUE TITLE
Delete an order

### ISSUE THREAD

**Issue by michaelgold** [orders-squad/orders#7]
**As a** developer   
**I need** to be able to delete an order
**So that** I can delete an order and make the manual change after customer cancel their order

**Assumptions:** 
* The cancel function will connect to the list all orders
* After the customer cancel the order, it will be deleted from the list of orders

**Acceptance Criteria:**
```
When a customer cancels an order 
Then the order will be deleted from the list of orders
```

**Comment by michaelgold** [comment]
I suggest rewording this story:

As a customer
I need to be able to cancel an order
So that I can get a refund

**Comment by EagleTongX** [comment]
I suppose this is a feature for the developer to delete the order after customer cancel it

**Comment by michaelgold** [comment]
I think the effort level here is a 2

**Comment by 75divisha** [comment]
I think the effort level here is a 2

**Comment by LihengGong** [comment]
If deleting an order only affects itself, then I agree with 2.
